src/event_pusher/mod_event_pusher_rabbit.erl

```diff
@@ -53,10 +54,13 @@
 start(Host, _Opts) ->
     application:ensure_all_started(amqp_client),
     application:ensure_all_started(worker_pool),
+    initialize_metrics(Host),
+    mongoose_metrics:init_subscriptions(),
```

Comment:
Are you **sure** you should have this call here? I mean isn't it intended to be called once per the whole system?

Reply:
I'll ask about that someone from MIM team.